expertsystem/io/__init__.py

```diff
@@ -24,6 +20,15 @@ def load_particle_collection(filename: str) -> ParticleCollection:
     )
 
 
+def load_pdg() -> ParticleCollection:
+    """Create a `.ParticleCollection` with all entries from the PDG.
+
+    PDG info is imported from the `scikit-hep/particle
+    <https://github.com/scikit-hep/particle/blob/master/README.rst>`_ package.
+    """
+    return _pdg.load_pdg()
```

Comment:
Why not just make the pdg module public and use `load_pdg()` directly from there?

Reply:
https://github.com/ComPWA/expertsystem/pull/190#discussion_r472318491